### PR TITLE
Send message to devlog channel on startup

### PR DESCRIPTION
**Pull request by Shivansh-007**
### Example
The place I scribbled is what the bot avatar would be.
ED89ABDE-E558-4527-BC2D-F67112414243

### About
On startup of a bot a message would be thrown in the configured dev log channel, the message lets us know that the bot has successfully connected after deployment.

### CODE REVIEW

**Comment by gustavwilliam** [comment]
In all PRs, I expect to see the following:
- A description of the feature or bug
- The general way you approached the issue
- Images, if applicable
- Other things that might be good to know about it, like edge cases and special considerations

Please update the description and I'll have a look.

**Comment by Shivansh-007** [comment]
> In all PRs, I expect to see the following:
> 
> * A description of the feature or bug
> * The general way you approached the issue
> * Images, if applicable
> * Other things that might be good to know about it, like edge cases and special considerations
> 
> Please update the description and I'll have a look.

@gustavwilliam Done!

**Review by vivax3794 — APPROVED** [review]
LGTM

File: bot/constants.py [hunk 1]
```diff
@@ -2,9 +2,15 @@
...
 PREFIX = os.getenv("PREFIX") or "!"
 TOKEN = os.getenv("TOKEN")
 
 # paths
 EXTENSIONS = pathlib.Path("bot/exts/")
 LOG_FILE = pathlib.Path("log/gurkbot.log")
+
+
+class Channels(NamedTuple):
+    devlog = int(os.environ.get("CHANNEL_DEVLOG", 622895325144940554))
```
Comment (gustavwilliam) [inline bot/constants.py:16]: The default channel ID doesn't seem quite right. The one here is actually for another server.

File: bot/bot.py [hunk 2]
```diff
@@ -32,3 +33,14 @@ def run(self) -> None:
     async def on_ready(self) -> None:
         """Ran when the bot has connected to discord and is ready."""
         logger.info("Bot online")
+        await self.startup_greeting()
+
+    async def startup_greeting(self) -> None:
+        """Announce presence to the devlog channel."""
+        embed = Embed(description="Connected!")
+        embed.set_author(
+            name="Gurk Bot",
```
Comment (gustavwilliam) [inline bot/bot.py:42]:
```suggestion
            name="Gurkbot",
```
This is one word.

File: bot/bot.py [hunk 3]
```diff
@@ -32,3 +33,14 @@ def run(self) -> None:
...
+            url="https://github.com/gurkult/gurkbot",
```
Comment (gustavwilliam) [inline bot/bot.py:43]: Let's put this in constants, so we have a single place to store the repo URL.

**Review by gustavwilliam — CHANGES_REQUESTED** [review]
Thanks. A few changes.

File: bot/bot.py (hunk 3 above)
Comment (Shivansh-007) [reply bot/bot.py:43]: Alright

File: bot/constants.py (hunk 1 above)
Comment (Shivansh-007) [reply bot/constants.py:16]: Ah yes, will fix it

**Review by gustavwilliam — APPROVED** [review]
Thanks! LGTM.

**Pull request merged**